chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json

```diff
@@ -49,13 +98,202 @@
+						"exec": [
+							"const developerSchema =",
+							"{",
+							"  \"developers\": [",
+							"    {",
+							"      \"address\": {",
+							"        \"addressId\": 0,",
+							"        \"city\": \"string\",",
+							"        \"country\": \"string\",",
+							"        \"line1\": \"string\",",
+							"        \"line2\": \"string\",",
+							"        \"state\": \"string\",",
+							"        \"zipcode\": \"string\"",
+							"      },",
+							"      \"contact\": {",
+							"        \"contactId\": 0,",
+							"        \"email\": \"string\",",
+							"        \"fullName\": \"string\",",
+							"        \"phoneNumber\": \"string\",",
+							"        \"title\": \"string\"",
+							"      },",
+							"      \"deleted\": false,",
+							"      \"developerCode\": \"string\",",
+							"      \"developerId\": 0,",
+							"      \"lastModifiedDate\": \"string\",",
+							"      \"name\": \"string\",",
+							"      \"selfDeveloper\": false,",
+							"      \"status\": {",
+							"        \"id\": 0,",
+							"        \"status\": \"string\"",
+							"      },",
+							"      \"statusEvents\": [",
+							"        {",
+							"          \"developerId\": 0,",
+							"          \"id\": 0,",
+							"          \"reason\": \"string\",",
+							"          \"status\": {",
+							"            \"id\": 0,",
+							"            \"status\": \"string\"",
+							"          },",
+							"          \"statusDate\": \"2020-10-09T17:08:51.022Z\"",
+							"        }",
+							"      ],",
+							"      \"transparencyAttestations\": [",
+							"        {",
+							"          \"acbId\": 0,",
+							"          \"acbName\": \"string\",",
+							"          \"attestation\": {",
+							"            \"removed\": false,",
+							"            \"transparencyAttestation\": \"string\"",
+							"          }",
+							"        }",
+							"      ],",
+							"      \"website\": \"string\"",
+							"    }",
+							"  ]",
+							"}",
+							"var jsonResponse = pm.response.json();",
+							"pm.test('developer schema is valid', function() {",
+							"  pm.expect(tv4.validate(jsonResponse, developerSchema)).to.be.true;  ",
+							"});",
+							"",
+							"pm.test(\"developer contact details should not include friendly name\", function () {",
+							"    pm.expect(pm.response.text()).to.not.include(\"friendlyName\");",
+							"});",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"protocolProfileBehavior": {
+				"disableBodyPruning": true
+			},
+			"request": {
+				"method": "GET",
+				"header": [
+					{
+						"key": "API-KEY",
+						"value": "{{apiKey}}",
+						"type": "text"
+					},
+					{
+						"key": "Content-Type",
+						"value": "application/json",
+						"type": "text"
+					},
+					{
+						"key": "Authorization",
+						"value": "{{authToken}}",
+						"type": "text",
+						"disabled": true
+					}
+				],
+				"url": {
+					"raw": "{{url}}/developers/{{developerId}}",
```

Comment:
I dont see {{developerId}} is being passed from anywhere

Reply:
This was something I started thinking about while writing tests that if we could go with parameterizing IDs instead of hard coding and thought of leaving this as an example here (it's in env. properties for now, which isn't quite the place but maybe we could come up with some other strategy for this (have that as a topic for weekly team meeting)

Reply:
This has been discussed and demo'd in our thursday's meeting. Example from demo never got merged. PR is https://github.com/chpladmin/chpl-api/pull/1007
This is good opportunity to get this in as part of this controller.

Reply:
I have added data driven tests for OCD-3443 ,which uses {{developerId}} values from the test data file for developer controller. Can this be used here also instead of having separate developerId parameter?